docs/Command-line-client.md

```diff
@@ -43,6 +43,19 @@ Syntax: `storm jar topology-jar-path class ...`
 
 Runs the main method of `class` with the specified arguments. The storm jars and configs in `~/.storm` are put on the classpath. The process is configured so that [StormSubmitter](javadocs/org/apache/storm/StormSubmitter.html) will upload the jar at `topology-jar-path` when the topology is submitted.
 
+When you want to ship other jars which is not included to application jar, you can pass them to `--jars` option with comma-separated string.
```

Comment:
That is not mandatory, but it only makes sense when that jar is not presented neither system provided nor included to application jar. If system or application jar provides that, that will be used. This is similar to precedence of classloading between system and application jar.